auth_backends.py

```diff
@@ -0,0 +1,53 @@
+from flask import request
+
+from webtodotxt import app
```

Comment:
Can you put this line at the very beginning of this file and without empty lines between these two imports?

Reply:
Addressed in latest commits.